src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static final String name = "rectangle";

```

Comment:
remove name fields from all the figures, it doesn't add value, Class.getSimpleName() would do it if necessary

Reply:
i dont understand this,please get mo info,tnx